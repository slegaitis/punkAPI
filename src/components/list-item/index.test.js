import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ListItem from './';

test('List Item component should render', () => {
	let item = {
		image_url: 'https://images.punkapi.com/v2/keg.png',
		name: 'Buzz',
		description:
			'BrewDog vs. Weihenstephan India Pale Weizen kicked off a new direction for our collaborations. We took a simple concept (by our standards) and teamed up with the oldest brewery in the world to brew a mash-up beer using the signature styles of both breweries. '
	};
	const component = shallow(<ListItem item={item} />);
	const tree = toJson(component);

	expect(tree).toMatchSnapshot();
});
