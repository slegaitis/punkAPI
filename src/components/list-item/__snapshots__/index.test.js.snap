// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Item component should render 1`] = `
<li
  className="list-item"
>
  <img
    alt="Buzz"
    src="https://images.punkapi.com/v2/keg.png"
  />
  <div
    className="description"
  >
    <h4>
      Buzz
    </h4>
    <p>
      BrewDog vs. Weihenstephan India Pale Weizen kicked off a new direction for our collaborations. We to
      ...
    </p>
  </div>
</li>
`;
